lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1312,10 +1312,15 @@ sub _failure_reason {
     my @failed_modules;
     my $modules = $self->modules;
     while (my $m = $modules->next) {
-        next if ($m->result eq INCOMPLETE);
-        push(@failed_modules, $m->name . ':' . $m->result);
+        if ($m->result eq FAILED) {
```

Comment:
should we also regard softfailed modules here? Or should we just go with what @aaannz proposed which is basically pushing all modules. But do we need to sort them?